internal/service/lightsail/container_service.go

```diff
@@ -0,0 +1,997 @@
+									},
+								},
+							},
+						},
+						"public_endpoint": {
+							Type:     schema.TypeList,
+							Optional: true,
+							MaxItems: 1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"container_name": {
+										Type:     schema.TypeString,
+										Required: true,
+									},
+									"container_port": {
+										Type:     schema.TypeInt,
+										Required: true,
+									},
+									"health_check": {
+										Type:     schema.TypeList,
+										Required: true,
+										MaxItems: 1,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"healthy_threshold": {
+													Type:     schema.TypeInt,
+													Optional: true,
+													Default:  2,
+												},
+												"interval_seconds": {
+													Type:         schema.TypeInt,
+													Optional:     true,
+													Default:      5,
+													ValidateFunc: validation.IntBetween(5, 300),
+												},
+												"path": {
+													Type:     schema.TypeString,
+													Optional: true,
+													Default:  "/",
+												},
+												"success_codes": {
+													Type:     schema.TypeString,
+													Optional: true,
+													Default:  "200-499",
+												},
+												"timeout_seconds": {
+													Type:         schema.TypeInt,
+													Optional:     true,
+													Default:      2,
+													ValidateFunc: validation.IntBetween(2, 60),
+												},
+												"unhealthy_threshold": {
+													Type:     schema.TypeInt,
+													Optional: true,
+													Default:  2,
+												},
+											},
+										},
+									},
+								},
+							},
+						},
+						"state": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"version": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+					},
+				},
+			},
+			"is_disabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+			"public_domain_names": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"certificate": {
+							Type:     schema.TypeSet,
+							Required: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"certificate_name": {
+										Type:     schema.TypeString,
+										Required: true,
+									},
+									"domain_names": {
+										Type:     schema.TypeList,
+										Required: true,
+										MinItems: 1,
+										Elem: &schema.Schema{
+											Type: schema.TypeString,
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+
+			// additional fields returned by lightsail container API
+			"arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"availability_zone": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
I would expect to see [created_at](https://docs.aws.amazon.com/lightsail/2016-11-28/api-reference/API_ContainerService.html#Lightsail-Type-ContainerService-createdAt), [isDisabled](https://docs.aws.amazon.com/lightsail/2016-11-28/api-reference/API_ContainerService.html#Lightsail-Type-ContainerService-isDisabled), [state_detail](https://docs.aws.amazon.com/lightsail/2016-11-28/api-reference/API_ContainerService.html#Lightsail-Type-ContainerService-stateDetail).

Reply:
Any other elements returned from the payload.

Reply:
`is_disabled` was there; added `created_at`. I omitted `state detail` because in here https://docs.aws.amazon.com/lightsail/2016-11-28/api-reference/API_ContainerServiceStateDetail.html, it says 
```
Note
The state detail is populated only when a container service is in a PENDING, DEPLOYING, or UPDATING state.
```
and for lightsail container service in terreform, all operations won't return unless the state moves out of the above states.